### ISSUE TITLE
missing z3c.form?

### ISSUE THREAD

**Issue by tkimnguyen** [plone/ploneorg.addonlisting#3]
With both Plone 5.0.4 and 5.0.6, and addonlisting git checkout, also as per https://community.plone.org/t/bin-instance-fg-gives-error-importerror-no-module-named-requests

```Traceback (most recent call last):
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/zinstance/parts/instance/bin/interpreter", line 285, in <module>
    exec(compile(__file__f.read(), __file__, "exec"))
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/Zope2-2.13.24-py2.7.egg/Zope2/Startup/run.py", line 76, in <module>
    run()
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/Zope2-2.13.24-py2.7.egg/Zope2/Startup/run.py", line 22, in run
    starter.prepare()
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/Zope2-2.13.24-py2.7.egg/Zope2/Startup/__init__.py", line 92, in prepare
    self.startZope()
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/Zope2-2.13.24-py2.7.egg/Zope2/Startup/__init__.py", line 268, in startZope
    Zope2.startup()
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/Zope2-2.13.24-py2.7.egg/Zope2/__init__.py", line 47, in startup
    _startup()
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/Zope2-2.13.24-py2.7.egg/Zope2/App/startup.py", line 118, in startup
    load_zcml()
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/Zope2-2.13.24-py2.7.egg/Zope2/App/startup.py", line 52, in load_zcml
    load_site()
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/Zope2-2.13.24-py2.7.egg/Zope2/App/zcml.py", line 46, in load_site
    _context = xmlconfig.file(site_zcml)
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/zope.configuration-3.7.4-py2.7.egg/zope/configuration/xmlconfig.py", line 651, in file
    context.execute_actions()
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/zope.configuration-3.7.4-py2.7.egg/zope/configuration/config.py", line 626, in execute_actions
    callable(*args, **kw)
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/plone.supermodel-1.3.0-py2.7.egg/plone/supermodel/model.py", line 130, in finalizeSchemas
    schema._SchemaClass_finalize()
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/plone.supermodel-1.3.0-py2.7.egg/plone/supermodel/model.py", line 90, in _SchemaClass_finalize
    adapter()
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/plone.supermodel-1.3.0-py2.7.egg/plone/supermodel/directives.py", line 176, in __call__
    model = loadFile(filename)
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/plone.supermodel-1.3.0-py2.7.egg/plone/supermodel/__init__.py", line 24, in loadFile
    parsed_model = parser.parse(path, policy=policy)
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/plone.supermodel-1.3.0-py2.7.egg/plone/supermodel/parser.py", line 70, in parse
    return _parse(source, policy)
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/plone.supermodel-1.3.0-py2.7.egg/plone/supermodel/parser.py", line 253, in _parse
    schema[fieldName]
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/plone.autoform-1.6.2-py2.7.egg/plone/autoform/supermodel.py", line 102, in read
    obj = resolveDottedName(widgetAttr)
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/plone.autoform-1.6.2-py2.7.egg/plone/autoform/utils.py", line 33, in resolveDottedName
    _dottedCache[dottedName] = resolve(dottedName)
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/zope.dottedname-3.4.6-py2.7.egg/zope/dottedname/resolve.py", line 38, in resolve
    __import__(used)
zope.configuration.config.ConfigurationExecutionError: <class 'plone.supermodel.parser.SupermodelParseError'>: No module named z3cform
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/zinstance/src/ploneorg.addonlisting/src/ploneorg/addonlisting/models/versioninfo.xml", line 11
    <schema>
  in:
  File "/Users/kim/PloneBuilds/Plone-5.0.6-unified/buildout-cache/eggs/plone.supermodel-1.3.0-py2.7.egg/plone/supermodel/configure.zcml", line 9.4-12.10
      <zcml:customAction
          handler=".model.finalizeSchemas"
          order="9999999"
          />```

**Comment by loechel** [comment]
plone.org is Plone 5.1 and it is only tested against 5.1a2.

could be an dependency error as we use a XML-Schema for Content Type definition and it uses collective.z3cform.datagridfield

**Comment by tkimnguyen** [comment]
Thx @loechel. Pinging @pavithirakc

**Comment by loechel** [comment]
I did fix it via 39b60dbd43e50a7583bba6f088a098352ebdda6f
